src/qos_enclave/Cargo.lock

```diff
@@ -1930,6 +2778,18 @@ version = "1.0.4"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "b6c140620e7ffbb22c2dee59cafe6084a59b5ffc27a8859a5f0d494b5d52b6be"
 
+[[package]]
+name = "utf8parse"
+version = "0.2.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "06abde3611657adf66d383f00b093d7faecc7fa57071cce2578660c9f1010821"
```

Comment:
This looks suspicious @cr-tk : https://crates.io/crates/utf8parse, and the github link goes to https://github.com/alacritty/vte, which isn't the right source code (package is "vte", not "utf8parse")

Reply:
This is fine, but was tricky to figure out: the `utf8parse` folder, which was part of the repo, isn't there anymore: https://github.com/alacritty/vte/pull/118/files